src/main/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializer.java

```diff
@@ -225,14 +225,19 @@ public DecodedResult deserialize(
       return result.get();
     }
     SchemaErrors.SchemaId schemaId = new SchemaErrors.SchemaId(context.getClusterId(), getSchemaIdFromRawBytes(bytes));
+    SchemaErrors.ConnectionId connectionId = new SchemaErrors.ConnectionId(context.getConnectionId());
```

Comment:
Again, why does `RecordDeserializer` know anything about `SchemaErrors.SchemaId` or `SchemaErrors.ConnectionId`? Those types should be _internal_ / _private_ to the `SchemaErrors` class.